doc/README_CN.md

```diff
@@ -24,7 +24,7 @@ Reference document
 * [APISIX 说明](../README_CN.md)
 * [架构设计](architecture-design-cn.md)
 * [压力测试](benchmark-cn.md)
-* [搭建开发环境](dev-manua-cnl.md)
+* [搭建开发环境](dev-manua-cn.md)
```

Comment:
`dev-manua-cn` is also a 404 link, please check again.

Reply:
`dev-manual-cn.md` correct address

Reply:
It should be `dev-manual-cn.md` in `d714b914542b9a2e904cd5f76e611adb57935eb7` branch.

Reply:
Just take a look at

`https://github.com/apache/incubator-apisix/blob/d714b914542b9a2e904cd5f76e611adb57935eb7/doc/README_CN.md`

and

`https://github.com/apache/incubator-apisix/blob/d714b914542b9a2e904cd5f76e611adb57935eb7/doc/`